Resources/migration.yml

```diff
@@ -145,6 +145,23 @@ CrateMedicalDefib: CrateMedical
 LeftLegSpider: null
 RightLegSpider: null
 WeaponShotgunEnforcerNonLethal: WeaponShotgunEnforcerRubber
+FloorPlanetAsphalt: FloorAsphalt
```

Comment:
You cant migrate tiles afaik :clueless:

Reply:
Oh, I swear I'd seen that already. Let me dig...

Reply:
hey, look what I found...
![image](https://github.com/DeltaV-Station/Delta-v-rebase/assets/107660393/03997483-8d64-46dc-8ca9-0346c1f95212)